Mathlib/Topology/Support.lean

```diff
@@ -110,6 +110,18 @@ theorem mulTSupport_mul [TopologicalSpace X] [Monoid α] {f g : X → α} :
     ((mulSupport_mul f g).trans (union_subset_union (subset_mulTSupport _) (subset_mulTSupport _)))
     (isClosed_closure.union isClosed_closure)
 
+theorem tsupport_smul_left [TopologicalSpace X] [Semiring R] [AddCommMonoid M] [Module R M]
+    [NoZeroSMulDivisors R M] (f : X → R) (g : X → M) : tsupport (f • g) ⊆ tsupport f := by
```

Comment:
This is strictly weaker than [tsupport_smul_subset_left](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Support.html#tsupport_smul_subset_left) right?

Reply:
Indeed, dropped.